tests/cli/test_deploy_cmd.py

```diff
@@ -26,8 +26,8 @@ def test_deployment_command_with_one_specified_contract(project):
     # weak assertion but not sure what to do here.
     assert 'Deploying Math' in result.output
     assert 'Deploying WithNoArgumentConstructor' not in result.output
-
-
+    
+    
```

Comment:
@Aviah 

Can you get your editor configure to remove trailing whitespace automatically.